[2025-06-16T19:10:30.735+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: diagnosis_pipeline.load_data manual__2025-06-16T19:10:29.725228+00:00 [queued]>
[2025-06-16T19:10:30.739+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: diagnosis_pipeline.load_data manual__2025-06-16T19:10:29.725228+00:00 [queued]>
[2025-06-16T19:10:30.739+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-06-16T19:10:30.747+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2025-06-16 19:10:29.725228+00:00
[2025-06-16T19:10:30.755+0000] {standard_task_runner.py:57} INFO - Started process 334 to run task
[2025-06-16T19:10:30.758+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'diagnosis_pipeline', 'load_data', 'manual__2025-06-16T19:10:29.725228+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/diagnosis_pipeline_dag.py', '--cfg-path', '/tmp/tmp3w5p97hf']
[2025-06-16T19:10:30.760+0000] {standard_task_runner.py:85} INFO - Job 2: Subtask load_data
[2025-06-16T19:10:30.805+0000] {task_command.py:416} INFO - Running <TaskInstance: diagnosis_pipeline.load_data manual__2025-06-16T19:10:29.725228+00:00 [running]> on host b95d2a4af95c
[2025-06-16T19:10:30.859+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='diagnosis_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-16T19:10:29.725228+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-16T19:10:29.725228+00:00'
[2025-06-16T19:10:30.860+0000] {load_data.py:8} INFO - Fetching dataset from UCI repository...
[2025-06-16T19:10:32.960+0000] {load_data.py:17} INFO - Saved X and y to results/
[2025-06-16T19:10:32.961+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-06-16T19:10:32.968+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=diagnosis_pipeline, task_id=load_data, execution_date=20250616T191029, start_date=20250616T191030, end_date=20250616T191032
[2025-06-16T19:10:33.016+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-16T19:10:33.032+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
